src/components/Dropdown/components/MultiValueContainer/MultiValueContainer.module.scss

```diff
@@ -1,6 +1,10 @@
 @import "../../../../styles/themes";
 @import "../../../../styles/typography";
 
+.chip-with-input-wrapper {
+  display: flex;
+  min-height: 30px;
```

Comment:
why we need `min-height` here?

Reply:
otherwise when the chip is too small the avatar become an ellipses instead of a circle